packages/trip-details/i18n/fr.yml

```diff
@@ -2,11 +2,12 @@
 otpUi:
   TripDetails:
     calories: "Calories dépensées : <strong>{calories, number, ::.}</strong>"
-    caloriesDescription: "La dépense calorique est calculée sur la base de
+    caloriesDescription: >
+      La dépense calorique est calculée sur la base de
       {walkMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} à pied et
       {bikeMinutes, plural, one {<strong># minute</strong> effectuée} other {<strong># minutes</strong> effectuées}} en vélo sur ce trajet.
       (d'après <dietaryLink>Dietary Guidelines for Americans 2005, page 16, Table 4</dietaryLink>)"
```

Comment:
Typo:  the trailing quote mark at end of this string.

Reply:
<img width="950" alt="Screen Shot 2021-12-03 at 1 12 50 PM" src="https://user-images.githubusercontent.com/64323/144673662-7d664f2c-6ed0-4866-ac05-e798c3754629.png">

Reply:
Thanks @fpurcell for spotting that one! Handled in https://github.com/opentripplanner/otp-ui/pull/321/commits/6850675cbd2c7a688d6e2a5ef50bac418f0fa03d.